app/pods/components/file-upload-minio/component.js

```diff
@@ -33,6 +33,15 @@ export default FileField.extend({
       }
     })
 
+    if(this.maxSize){
```

Comment:
I think we should just give a default value to maxSize for `file-upload-minio` component. We never allow unlimited size uploads and backend checks will ensure this anyways. It makes sense to use a reasonable default here and skip the "existence" check